### PR TITLE
Readme: replace deprecated multirust with rustup.rs

### PR DESCRIPTION
**Pull request by jirutka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ Currently, it **must** be built using a nightly release of the Rust compiler.
 Nightly releases can be found at the bottom of the
 [Rust install page](http://www.rust-lang.org/install.html)
 or installed using the `--channel=nightly` option to
-[rustup](https://github.com/rust-lang/rustup).
+[rustup.sh](https://github.com/rust-lang/rustup).
```
Comment (murarth) [inline README.md]: You've linked the wrong rustup project here. It should be `[rustup.rs](https://github.com/rust-lang-nursery/rustup.rs)` as below.

**Comment by murarth** [comment]
There's an issue affecting rustup-rs right now that prevents easily installing the nightly compiler. I'll most likely merge this PR once that issue is resolved.

**Comment by jirutka** [comment]
What issue? `rustup toolchain install nightly` works. However, there’s different issue, with locating Rust’s shared libraries. That reminds my that I should report it (if it’s not already).

**Comment by jirutka** [comment]
Hm, already there (https://github.com/rust-lang-nursery/rustup.rs/issues/350) and they don’t support dynamically linked executables. 😿

**Comment by murarth** [comment]
It wasn't working for me last night. It seems to be a server-side issue, so its effect is intermittent. rust-lang-nursery/rustup.rs#524

**Comment by murarth** [comment]
The issue with rustup.rs and the latest nightly is now irrelevant as Rusti will build only with a nightly no later than `2016-08-01`. The README has been updated to reflect this and to suggest rustup.rs as the preferred method of installation of `nightly-2016-08-01`.

**Pull request closed without merging**